src/seabirdscientific/conversion.py

```diff
@@ -61,10 +70,13 @@
 KELVIN_OFFSET_25C = 298.15
 OXYGEN_MLPERL_TO_MGPERL = 1.42903
 OXYGEN_MLPERL_TO_UMOLPERKG = 44660
-ITS90_TO_IPTS68 = 1.00024  # taken from https://blog.seabird.com/ufaqs/what-is-the-difference-in-temperature-expressions-between-ipts-68-and-its-90/
+# taken from https://blog.seabird.com/ufaqs/what-is-the-difference-in-temperature-expressions-between-ipts-68-and-its-90/
+ITS90_TO_IPTS68 = 1.00024
 UMNO3_TO_MGNL = 0.014007
```

Comment:
It might be helpful to some to add a comment as to what UMNO3_TO_MGNL is. I assume its for microMoles of Nitrogen/Nitrate? to milligrams Nitrogen/Nitrate? per liter.